src/custom/api/gnosisProtocol/api.ts

```diff
@@ -21,6 +21,7 @@ import { FeeQuoteParams, PriceInformation, PriceQuoteParams, SimpleGetQuoteRespo
 
 import { DEFAULT_NETWORK_FOR_LISTS } from 'constants/lists'
 import * as Sentry from '@sentry/browser'
+import { constructSentryError } from 'utils/sentry'
```

Comment:
nit: `utils/error` or `utils/logging` just to abstract that is sentry?

Reply:
done